src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  FC,
+  memo,
+  useCallback,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoTitleField } from '../TodoTitleField/TodoTitleFields';
+
+  type Props = {
+    todo: Todo,
+    isDeleting: boolean,
+    onDeleteTodo: (todoId: number) => void,
+    updateTodo: (
+      todoId: number,
+      fieldsToUpdate: Partial<Pick<Todo, 'title' | 'completed'>>
+    ) => Promise<void>,
```

Comment:
please check the function from parent component. is should be `void`

Reply:
done